Libraries/Animated/src/AnimatedImplementation.js

```diff
@@ -1908,17 +2030,23 @@ var event = function(
         typeof recEvt === 'object',
         'Bad event of type ' + typeof recEvt + ' for key ' + key
       );
-      for (var key in recMapping) {
-        traverse(recMapping[key], recEvt[key], key);
+      for (const mappingKey in recMapping) {
+        traverse(recMapping[mappingKey], recEvt[mappingKey], mappingKey);
       }
     };
-    argMapping.forEach((mapping, idx) => {
-      traverse(mapping, args[idx], 'arg' + idx);
-    });
-    if (config && config.listener) {
-      config.listener.apply(null, args);
-    }
-  };
+  }
+}
+
+var event = function(
+  argMapping: Array<?Mapping>,
+  config?: ?EventConfig,
+): any {
```

Comment:
Right now it works with proptypes we return the handler in AnimatedProps (https://github.com/facebook/react-native/pull/9253/files#diff-f048d92ca0be679bc38d38147b311100R1402) so I guess we're doing something like 2).